index.html

```diff
@@ -32,47 +32,52 @@
+            const positionOnGlobe = { longitude: 2.3465, latitude: 48.88, altitude: 25000000 };
 
             // iTowns namespace defined here
-            var viewerDiv = document.getElementById('viewerDiv');
-            var menuGlobe = new GuiTools(itowns.viewer,'menuDiv');
+            const viewerDiv = document.getElementById('viewerDiv');
+            const menuGlobe = new GuiTools(itowns.viewer, 'menuDiv');
             itowns.viewer.createSceneGlobe(positionOnGlobe, viewerDiv);
 
-            let wgs84TileLayer = {
+            const wgs84TileLayer = {
                 protocol: 'tile',
-                id: 'wgs84'
+                id: 'wgs84',
             };
 
-			var urlImageryLayers = [], urlElevationLayers = [];
+            const urlImageryLayers = [];
+            const urlElevationLayers = [];
 
             urlImageryLayers.push('examples/layers/JSONLayers/Ortho.json');
             urlImageryLayers.push('examples/layers/JSONLayers/OrthosCRS.json');
             urlImageryLayers.push('examples/layers/JSONLayers/ScanEX.json');
             urlImageryLayers.push('examples/layers/JSONLayers/Region.json');
 
-		  	urlElevationLayers.push('examples/layers/JSONLayers/IGN_MNT.json');
+            urlElevationLayers.push('examples/layers/JSONLayers/IGN_MNT.json');
             urlElevationLayers.push('examples/layers/JSONLayers/IGN_MNT_HIGHRES.json');
 
             itowns.viewer.addGeometryLayer(wgs84TileLayer);
-            let imageryPromise = itowns.viewer.addImageryLayersFromJSONArray(urlImageryLayers);
-            let elevationPromise = itowns.viewer.addElevationLayersFromJSONArray(urlElevationLayers);
+            const imageryPromise = itowns.viewer.addImageryLayersFromJSONArray(urlImageryLayers);
+            const elevationPromise = itowns.viewer.addElevationLayersFromJSONArray(urlElevationLayers);
 
-            imageryPromise.then((layersColors) => menuGlobe.addImageryLayersGUI(layersColors));
-            elevationPromise.then((layersElevations) => {menuGlobe.addElevationLayersGUI(layersElevations);});
+            imageryPromise.then((layersColors) => { menuGlobe.addImageryLayersGUI(layersColors); });
+            elevationPromise.then((layersElevations) => { menuGlobe.addElevationLayersGUI(layersElevations); });
 
             Promise.all([imageryPromise, elevationPromise]).then(() => itowns.viewer.init());
 
-            menuGlobe.addGUI('RealisticLighting',false,function(newValue) {itowns.viewer.setRealisticLightingOn(newValue);});
+            menuGlobe.addGUI('RealisticLighting', false,
+                (newValue) => { itowns.viewer.setRealisticLightingOn(newValue); });
 
-           	itowns.viewer.addEventListener('globe-loaded', function() {
-            	console.log("Globe Loaded");
-        	});
-        </script>
+            itowns.viewer.addEventListener('globe-loaded', () => {
+                /* eslint-disable no-alert, no-console */
```

Comment:
Yes, there are errors with `console.log`, this is one of the first rules we added. I prefer like this